api/v1/db/oracledb/games-dao.js

```diff
@@ -112,4 +112,28 @@ const postGame = async (body) => {
   }
 };
 
-module.exports = { getGames, getGameById, postGame };
+/**
+ *
+ * @param {id} developerId id of developer record to check existance
```

Comment:
Param type should be `{string}`. Can you also add `@summary` to this function?

Ref: https://jsdoc.app/tags-type.html